packages/lesswrong/components/posts/PostsPage/PostsPage.tsx

```diff
@@ -556,7 +562,7 @@ const PostsPage = ({post, eagerPostComments, refetch, classes}: {
     </>}
     {/* Header/Title */}
     <AnalyticsContext pageSectionContext="postHeader">
-      <div className={classes.title}>
+      <div className={classNames(classes.title, {[classes.titleWithMarket] : highlightMarket(getMarketInfo(post))})}>
```

Comment:
too many levels of nesting inside of a style class application for my taste - I'd pull `highlightMarket(getMarketInfo(post))` out to a variable declaration

Reply:
(Actually it looks like you're using `getMarketInfo(post)` in two places, so maybe that instead)